### PR TITLE
feat(src/extension/_locales/en): better messages

### PR DESCRIPTION
**Pull request by MarquiseRosier**
make error messages, more clearly show what users must do to make Sidekick work when invalid url error is given

## Related Issues
#169 

Thanks for contributing!

### CODE REVIEW

File: src/extension/_locales/de/messages.json [hunk 1]
```diff
@@ -117,7 +117,7 @@
     "message": "Ungültige Teilen-URL"
   },
   "config_invalid_giturl": {
-    "message": "Ungültige Repository-URL"
+    "message": "Ungültige Repository-URL, muss beginnen mit https://github.com"
```
Comment (rofe) [inline src/extension/_locales/de/messages.json:120]: I think I would change this to 
```suggestion
    "message": "Ungültige Repository-URL, muss mit https://github.com beginnen"
```

**Review by rofe — CHANGES_REQUESTED** [review]
@kptdobe could you give the French translation a look?

File: src/extension/_locales/fr/messages.json [hunk 2]
```diff
@@ -117,7 +117,7 @@
     "message": "URL de partage incorrecte"
   },
   "config_invalid_giturl": {
-    "message": "URL du code dans GitHub incorrecte"
+    "message": "URL de référentiel non valide, doit commencer par https://github.com"
```
Comment (kptdobe) [inline src/extension/_locales/fr/messages.json:120]: I would simplify the message to: 
```suggestion
    "message": "L'URL du projet doit commencer par https://github.com"
```

**Comment by rofe** [comment]
@MarquiseRosier are you going to finish this PR?

**Comment by rofe** [comment]
Closing due to inactivity.

**Pull request closed without merging**